final/discuss-recommend.tex

```diff
@@ -1,8 +1,26 @@
 \chapter{Discussion and Recommendations}
 \label{cha:disc-recomm}
 
+In this chapter the achieved results will be discussed, as well as some tasks
+that were not achieved. In \cref{cha:evaluation} we have already discussed how
```

Comment:
So we won't discuss the redesign because of the multithread issue etc? Fine with me, just checking.